website/docs/intro.mdx

```diff
@@ -34,12 +34,12 @@ Some of the things Taqueria can do for you include:
 
 Taqueria is an extensible, open source plugin based framework and task runner for Tezos development
 
-In more practical terms, Taqueria is a task runner that you can use from the command line or the VS Code command palette to quickly accomplish things like starting a sandbox or originating a contract. Taqueria comes with a few basic tasks, but relies on plugins to add tasks that provide complex and domain specific functionality
+In more practical terms, Taqueria is a task runner that you can use from the command line order VS Code to quickly accomplish things like starting a sandbox or originating a contract. Taqueria comes with a few basic tasks, but relies on plugins to add tasks that provide complex and domain specific functionality
```

Comment:
In more practical terms, Taqueria is a task runner and source file manager that can be used from the command line or via VS Code to quickly accomplish things like starting a sandbox and originating a contract. Taqueria comes with a few basic tasks, but relies on plugins to add tasks that provide complex and domain specific functionality.